module/actor/maneuver.js

```diff
@@ -1,5 +1,5 @@
+import { TokenGURPS } from '../token/index.js'
 import * as settings from '../../lib/miscellaneous-settings.js'
```

Comment:
Again, a comment you don't have to fix immediately: Following along with the comments on `global.d.ts`, we should be consistent in our naming. So if we import `* as Settings` in one place, we should do that everywhere and not vary it (such as `* as settings`).

Reply:
Resolved in latest commit.